job-service-postgres-container/pom.xml

```diff
@@ -120,7 +123,7 @@
                         <!-- Run old version of jobservice postgres container against postgres database to install jobservice schema. -->
                         <image>
                             <alias>job-service-postgres-old</alias>
-                            <name>${dockerHubPublic}/jobservice/job-service-postgres:3.0</name>
+                            <name>${dockerHubPublic}/jobservice/job-service-postgres:2</name>
```

Comment:
Yes, as the migration script has been adapted to any version (we wanted to avoid a breaking change back then), we re trying to check it from the oldest possible version (couldn't find 1.0)